Directory.Packages.props

```diff
@@ -20,13 +22,15 @@
     <PackageVersion Include="SonarAnalyzer.CSharp" Version="10.15.0.120848" />
     <PackageVersion Include="StyleCop.Analyzers" Version="1.2.0-beta.556" />
     <PackageVersion Include="Swashbuckle.AspNetCore" Version="9.0.6" />
+    <PackageVersion Include="System.Drawing.Common" Version="10.0.0" />
+    <PackageVersion Include="System.IdentityModel.Tokens.Jwt" Version="8.14.0" />
```

Comment:
`System.Drawing.Common` should not be required and `System.IdentityModel.Tokens.Jwt` doesn't appear to be referenced in a csproj-file, but I could be wrong